meta-lmp-bsp/dynamic-layers/stm-st-stm32mp/recipes-bsp/trusted-firmware-a/tf-a-fio_%.bbappend

```diff
@@ -0,0 +1,30 @@
+# Define config for each TF_A_CONFIG
```

Comment:
This is interesting as the options are all specific to st boards, and since the main recipe is under meta-lmp-base, just adding meta-st (which is done by default) is enough for this append to be parsed (for any machine).

We should probably have a common tf-a-fio-st.inc or similar under meta-lmp-bsp (no dynamic-layers) and only include if stm32mp1common.